app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectInformationActivity.kt

```diff
@@ -567,6 +568,7 @@ class ProjectInformationActivity : AppCompatActivity() {
 
     override fun onPrepareOptionsMenu(menu: Menu): Boolean {
         menu.findItem(R.id.waitingListButton)?.isVisible = (userId == projectVar.authorId)
+        menu.findItem(R.id.editButton)?.isVisible = (userId == projectVar.authorId)
```

Comment:
Using an intermediate boolean could reduce code dupplication (but it is not important)
```Suggestion
val isOwnProject: Boolean = (userId == projectVar.authorId)
menu.findItem(R.id.waitingListButton)?.isVisible = isOwnProject
menu.findItem(R.id.editButton)?.isVisible = isOwnProject
```